### DISCUSSION TITLE
SDR Trunk

### DISCUSSION THREAD

**Discussion by JasleenKaur2308** [DSheirer/sdrtrunk#1652] [Q&A]
Can we listen to normal FM in SDR Trunk ? If yes , then how can we listen to Wideband FM signals in SDR Trunk?

Link for reference - https://github.com/DSheirer/sdrtrunk/wiki

**Comment by DSheirer** [comment]
The widest channel bandwidth supported for FM decode is 25 kHz.

**Reply by JasleenKaur2308** [reply]
Thank you so much for your reply .

**Reply by JasleenKaur2308** [reply]
Hello ! i have one more query , what if i want to read a DMR file in SDR Trunk ? What the steps for the following and how i will use the tuner?
